SimG4Core/Application/plugins/OscarProducer.cc

```diff
@@ -102,6 +102,8 @@ OscarProducer::OscarProducer(edm::ParameterSet const & p)
   produces<edm::PSimHitContainer>("MuonCSCHits");
   produces<edm::PSimHitContainer>("MuonRPCHits");
   produces<edm::PSimHitContainer>("MuonGEMHits");
+  produces<edm::PSimHitContainer>("MuonME0Hits");
+
```

Comment:
Please make the same change to OscarMTProducer too (that is already the default in 75X).